shared/models/list.yml

```diff
@@ -1,25 +1,24 @@
-List:
-  type: object
-  description: Multiple items returned in order
-  properties:
-    object:
-      type: string
-      description: What kind of resource does this list contain?
-      example: address
+type: object
+description: Multiple items returned in order
+properties:
+  object:
+    type: string
+    description: What kind of resource does this list contain?
+    example: address
 
-    next_url:
-      type: string
-      description: Url of next page of items in list.
-      nullable: true
-      example: https://api.lob.com/v1/addresses?limit=2&after=eyJkYXRlT2Zmc2V0IjoiMjAxOS0wOC0wN1QyMTo1OTo0Ni43NjRaIiwiaWRPZmZzZXQiOiJhZHJfODMwYmYwZWFiZGFhYTQwOSJ9
+  next_url:
+    type: string
+    description: Url of next page of items in list.
+    nullable: true
+    example: https://api.lob.com/v1/addresses?limit=2&after=eyJkYXRlT2Zmc2V0IjoiMjAxOS0wOC0wN1QyMTo1OTo0Ni43NjRaIiwiaWRPZmZzZXQiOiJhZHJfODMwYmYwZWFiZGFhYTQwOSJ9
 
-    previous_url:
-      type: string
-      description: Url of previous page of items in list.
-      nullable: true
-      example: null
+  previous_url:
+    type: string
+    description: Url of previous page of items in list.
+    nullable: true
+    example: null
```

Comment:
An example URL would be nice here (even though the url structure is arguably opaque).

Reply:
It isn't so much that the structure is arguably opaque as that there's no simple way to override an incorrect example, at least in OpenAPI v3.0. A lot of work went into examples in the upgrade between 3.0 and 3.1.

We'll need to wait until we've either got an extension in place or are on 3.1 to have examples for this one, unfortunately.